src/hardware/cpe-ubiquiti.md

```diff
@@ -0,0 +1,113 @@
+# CPE - Ubiquiti
+
+This document covers setup for the following devices. Other devices may be similar.
+
+- [Loco5AC](https://www.ui.com/airmax/nanostation-ac/)
+- [LBE‑5AC‑23](https://www.ui.com/airmax/litebeam-ac/)
+
+## What you will need
+
+You will need to collect the following information:
+
+- SSID of the supernode antenna you wish to connect to
+- Your assigned node name
+- Your assigned IP address, Netmask, Gateway
+- A computer with WiFi for management radio method or a computer with a network port for cable method.
+
+
+## Connecting to the CPE for the first time.
+
+### If the device has a management radio: Over WiFi
```

Comment:
```suggestion
```suggestion
### Connection using the Management Radio

Most devices come with a second `Management Radio` that is active for a short period of time after power on.  While active it creates a wireless access point dedicated to configuration of your device. You can connect to it using your laptop's or phone's wireless capability.
```
```